fedimint-server/src/net/peers.rs

```diff
@@ -82,36 +79,50 @@ struct PeerConnectionStateMachine<M> {
 }
 
 /// Calculates delays for reconnecting to peers
-/// The default values are in the order of seconds
 #[derive(Debug, Clone, Copy)]
 pub struct DelayCalculator {
-    scaling_factor: f64,
+    min_retry_duration_ms: u64,
+    max_retry_duration_ms: u64,
 }
 
 impl DelayCalculator {
-    fn new() -> Self {
-        Self {
-            scaling_factor: 1.0,
-        }
-    }
+    /// Production defaults will try to reconnect fast but then fallback to
+    /// larger values if the error persists
+    const PROD_MAX_RETRY_DURATION_MS: u64 = 10_000;
+    const PROD_MIN_RETRY_DURATION_MS: u64 = 10;
+
+    /// For tests we don't want low min/floor delays because they can generate
```

Comment:
I thought we wanted some short retries as it speeds things up in the presence of artificially introduced failures?

I don't think we mind a bit of noise. The problem was that that we had walls (> 100, maybe >1000) of these lines.

Reply:
With respect to test execution speed I worry more about the max/ceiling limits than the min/floor limits.  But of course lower min/floor limits would be better.

Here are some execution with different parameters, `exp_MIN_MAX.log`, lower `MIN/FLOOR` values tend to create more logs, although the actual number is highly variable:

```
❯ grep -c 'Replacing existing connection' exp_10_10000.log 
797

❯ grep -c 'Replacing existing connection' exp_100_10000.log 
534

❯ grep -c 'Replacing existing connection' exp_2000_10000.log 
228

❯ grep -c 'Replacing existing connection' exp_2000_10000.log 
262

❯ grep -c 'Replacing existing connection' exp_10_5000.log 
1114

❯ grep -c 'Replacing existing connection' exp_2000_5000.log 
150

❯ grep -c 'Replacing existing connection' exp_2000_5000.log 
156

❯ grep -c 'Replacing existing connection' exp_4000_5000.log 
276

❯ grep -c 'Replacing existing connection' exp_10_15000.log 
1474

❯ grep -c 'Replacing existing connection' exp_2000_15000.log 
120

❯ grep -c 'Replacing existing connection' exp_10_30000.log   
773

❯ grep -c 'Replacing existing connection' exp_2000_30000.log
130
```